templates/contacts.json

```diff
@@ -31,38 +55,7 @@
       "dataset": "{{@ system @}}-{{@ datatype @}}-transform",
       "type": "dataset"
     },
-    "transform": [
```

Comment:
I honestly have no idea why this was in the share... It's automatically in transform